src/main/java/Cost.java

```diff
@@ -0,0 +1,25 @@
+public class Cost {
+
+    private static final int MINIMUM_PRICE = 1000;
+    private static final int LOTTO_PRICE = 1000;
+    private final int cost;
+
+    public Cost(int cost) {
+        this.cost = cost;
+        validate(cost);
+    }
+
+    private void validate(int cost) {
+        if (cost < MINIMUM_PRICE) {
+            throw new IllegalArgumentException("로또 금액이 부족합니다.");
```

Comment:
금액이 얼마인지 예외 메시지에 보인다면 에러를 수정하기가 조금 더 쉽지 않을까요?😃
```suggestion
            throw new IllegalArgumentException("로또 금액이 부족합니다. cost: " + cost);
```

Reply:
금액도 출력하도록 변경했습니다. [a971c83](https://github.com/next-step/java-lotto-kakao/pull/12/commits/a971c8318e9c657cb37f18e4a9d556bfc535f997)